### PR TITLE
Rocko backports v2019.07

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: meta-mender-qemu/scripts/docker/entrypoint.sh [hunk 1]
```diff
@@ -24,11 +24,16 @@ if [ -f /mnt/config/artifact-verify-key.pem ]; then
...
 if [ ! -e /mender-setup-complete ]; then
     ./setup-mender-configuration.py --img="$DISK_IMG" \
                                     --server-url=$SERVER_URL \
-                                    --tenant-token=$TENANT_TOKEN $CONFIG_ARGS
+                                    --tenant-token=$TENANT_TOKEN $CONFIG_ARGS \
+                                    --docker-ip="$DOCKER_IP"
     touch /mender-setup-complete
 fi
 
+export QEMU_NET_HOSTFWD=",hostfwd=tcp::80-:80"
```
Comment (kacf) [inline meta-mender-qemu/scripts/docker/entrypoint.sh]: Don't you also need 021ed34f5d049fcd4f8e4f43b9c620c9a53182e0?

**Comment by lluiscampos** [comment]
New builds after cherry-picks:
* 1.7.1 in [Jenkins 3954](https://mender-jenkins.mender.io/job/mender-builder/3954/)
* 2.0.1 in [Jenkins 3955](https://mender-jenkins.mender.io/job/mender-builder/3955/)

**Comment by lluiscampos** [comment]
Also, please look at commit b866cb3. The original one did not apply on rocko, so I basically re-implemented it.

**Comment by lluiscampos** [comment]
Same as PR #775,

I updated the back-port with latest changes released in `thud-v2019.08` except for:
* Allow default bootloader to be overridden  8d81c212a4d61d3b9bb5ddc8ee7dff955700d710 (and related commits)
* mender: Setup Live installer with HDDIMG 698ef94741289131ff18cea2d71bda412c7a4e17

Builds in progress:
* 1.7.1 https://mender-jenkins.mender.io/job/mender-builder/4313/
* 2.0.1 https://mender-jenkins.mender.io/job/mender-builder/4314/
* 2.1.0b1 https://mender-jenkins.mender.io/job/mender-builder/4315/

**Comment by lluiscampos** [comment]
@kacf Except for raspberrypi config, all other builds are green for the three versions. On `rocko` raspberrypi was failing already.

**Pull request merged**